types/pdfmake/test/pdfmake-module-server-tests.ts

```diff
@@ -20,10 +21,17 @@ const dd: TDocumentDefinitions = {
     content: 'Hello world!',
 };
 
+const customTableLayouts: {[key: string]: TableLayout} = {
+    customLayout: {
+        hLineColor: 'red',
+        vLineColor: () => 'red',
+    }
+};
+
 const options: BufferOptions = {
     fontLayoutCache: true,
     bufferPages: true,
-    tableLayouts: 'noBorders',
+    tableLayouts: customTableLayouts,
```

Comment:
Please don't change existing tests that are correct, rather add new tests to cover as much of the new definitions as possible.

Reply:
from my understanding, even though it's passing, I don't think that this test is correct: tableLayouts should be an object containing user-defined layouts, providing a pre-defined layout is pointless.
what do you think ?

Reply:
Oh you're right. I didn't notice this was in BufferOptions. Good catch